src/ngraph/runtime/gpu/gpu_backend.hpp

```diff
@@ -29,7 +29,7 @@ namespace ngraph
         {
             static size_t alignment = 64;
 
-            class GPU_ExternalFunction;
+            class GPU_CompiledFunction;
```

Comment:
Since you are changing the name change it to `GPUCompiledFunction` to be consistent.